Makefile

```diff
@@ -11,38 +11,68 @@ package_name=$(app_name)
 cert_dir=$(HOME)/.nextcloud/certificates
 version+=master
 
-all: appstore
+all: dev-setup build-js-production
+
+dev-setup: clean-dev npm-init
+
+dependabot: dev-setup npm-update build-js-production
```

Comment:
Is this run by _dependabot_ itself or is it meant to be run manually? In any case, I guess that `compile-handlebars-templates` and `bundle-simplewebrtc` should be added as dependencies of the step, given that after running `npm update` the Handlebars templates and the bundled SimpleWebRTC could change?

Reply:
No, it is run by me to update all in one PR.... jut legacy and we can remove it in the future I guess once we update all from the dependabot PRs as we dont need to compile and therefor combine anymore.

I just like to have a quick way to update all and build it automatically.

Reply:
they are part of `build-js-production` so it's done already

Reply:
Right, I removed it there....